python/distant_vfx/shotgun.py

```diff
@@ -69,12 +69,12 @@ def create_shot(self, shot_code, sequence_id, status='wtg', other_data=None):
 
         # If additional data is provided, merge into the original data dict. Note this will override any duplicate keys.
         if other_data is not None:
-            data = {**data, **other_data}
+            data.update(other_data)
```

Comment:
Another py3 pattern I haven't come across before, but I think you're just trying to update the existing data dictionary with data from other_data.

If that's correct, this is much more efficient and works on py2 too.

Reply:
Yeah that's right, I was following [this method](https://stackoverflow.com/questions/38987/how-do-i-merge-two-dictionaries-in-a-single-expression-in-python-taking-union-o). I think the difference is it returns a new dict rather than modifying it in place, but I guess it doesn't really matter here.

Reply:
Yeah ok cool, that is 100% correct, the efficiencies come from not returning a brand new one, and you are replacing it, so we don't need to worry about modifying it in place :)
That's a good distinction to note though